### PR TITLE
Helmet rendertron. Fixed #399

### LINKED ISSUE #399: v2: allow to customize meta tags
It would be great if depending on a page meta-tags could be customized.

Use cases: load blog post summary / speaker short bio when URL for speaker/blog post is shared.

### PR DESCRIPTION
**Pull request by ozasadnyy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
   <!-- G+ sharing meta data -->
   <meta itemprop="name" content="{$ title $}">
   <meta itemprop="description" content="{$ description $}">
-  <meta itemprop="image" content="{$ url $}images/social-share.jpg">
+  <meta itemprop="image" content="{$ image if image.startsWith('http') else (url + image) $}">
```
Comment (sophieH29) [inline index.html]: It's not good to check url like this.  How about using regex to validate if it is valid URL?

File: src/elements/polymer-helmet.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            type: String,
+            value: '{$ title $}',
+          },
+          _defaultDescription: {
+            type: String,
+            value: '{$ description $}',
+          },
+          _defaultImage: {
+            type: String,
+            value: '{$ image if image.startsWith("http") else (url + image) $}',
```
Comment (sophieH29) [inline src/elements/polymer-helmet.html]: Same here

File: src/elements/polymer-helmet.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      }
+
+      static get observers() {
+        return [
+          '_setMetaInfo(active, title, description, image, label1, data1, label2, data2)',
+        ];
+      }
+
+      _setMetaInfo(active, title, description, image, label1, data1, label2, data2) {
+        if (active) {
```
Comment (sophieH29) [inline src/elements/polymer-helmet.html]: you can add at the beginning

title = title || this._defaultTitle;
description = description || this._defaultDescription;

....

So you wouldn't repeat this checks

File: src/pages/schedule-page.html [hunk 4]
```diff
@@ -184,6 +190,10 @@
           });
         }
       }
+
+      _setHelmetData(active, isDialogOpened) {
```
Comment (sophieH29) [inline src/pages/schedule-page.html]: where do you get this 'active' value?

File: index.html (hunk 1 above)
Comment (ozasadnyy) [reply index.html]: I am not checking if it's valid url or not. What I do is to check if it's an absolute path or relative

File: src/pages/schedule-page.html (hunk 4 above)
Comment (ozasadnyy) [reply src/pages/schedule-page.html]: From `lazy-pages`

**Pull request merged**